src/containers/Search/CloneModel.js

```diff
@@ -0,0 +1,160 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { loadMyProjectsAction } from 'store/actions/project';
+import { clonePlaylist, cloneActivity } from 'store/actions/search';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Accordion, Card, Button } from 'react-bootstrap';
+import Swal from 'sweetalert2';
+
+export default function LTIProjectShared(props) {
+  const [activeProject, setActiveProject] = useState('');
+  const [activePlaylist, setActivePlaylist] = useState('');
+  // const [activeactivity, setactiveactivity] = useState("");
+  // const [activeactivitydefault, setactiveactivitydefault] = useState(true);
+  const [allproject, setAllproject] = useState(null);
+  const [currentProject, setCurrentProject] = useState();
+  const [currentPlaylist, setCurrentPlaylist] = useState();
+  const dispatch = useDispatch();
+  const allProjects = useSelector((state) => state);
+  useEffect(() => {
+    dispatch(
+      loadMyProjectsAction(),
+    );
+  }, [dispatch]);
+
+  useEffect(() => {
+    allProjects.project.projects && setAllproject(allProjects.project.projects);
```

Comment:
Please do not use this kind of approach
Use if condition

Reply:
done